pegasus/sites.v3/code.org/views/inspirational_videos.md

```diff
@@ -0,0 +1,41 @@
+<div style="float:left; padding:10px;" class="col-50">
```

Comment:
@caleybrock this would probably be more readable in haml.  we might also be able to use an array and an iterator to remove duplicated code, too.